### PR TITLE
Add instruction how to setup flatpaks

### PR DESCRIPTION
**Pull request by ShockwaveNN**

### CODE REVIEW

**Comment by agolybev** [comment]
Hi, can anybody answer if we can add a `REDME.md` file with flatpack build instruction into the repo? 
I think it would be helpful for our developers and testers. But there aren't any readme-files in other flatpak repositories.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+   ```shell script
+   flatpak install flathub org.freedesktop.Platform//19.08 \
+                   org.freedesktop.Sdk//19.08 \
+                   io.atom.electron.BaseApp//19.08
+   ```
+
+4. Build and install the application
+
+   ```shell script
+   sudo flatpak-builder build org.onlyoffice.desktopeditors.json --force-clean --install
```
Comment (bilelmoussaoui) [inline README.md:51]: Never run flatpak with sudo. It uses polkit to ask for privileges when they are needed

Comment (ShockwaveNN) [reply README.md]: Thanks, fixed it

**Comment by bilelmoussaoui** [comment]
> Hi, can anybody answer if we can add a `REDME.md` file with flatpack build instruction into the repo?
> I think it would be helpful for our developers and testers. But there aren't any readme-files in other flatpak repositories.

The reason we ask developers to avoid adding a README file is to not duplicate documentations that might end up outdated in various places. If that helps the developers, feel free to merge it

**Pull request merged**